src/platforms/linux-openwrt.js

```diff
@@ -132,6 +132,39 @@ function uciCommit(key) {
   return proc.status === 0;
 }
 
+/**
+ * Get DHCP server status.
+ *
+ * @returns {boolean} Boolean indicating whether or not DHCP is enabled.
+ */
+function getDhcpStatus() {
+  const result = uciGet('dhcp.lan.ignore');
+  if (!result.success) {
+    return false;
+  }
+
+  return result.value === '0';
+}
+
+/**
+ * Set DHCP server status.
+ *
+ * @param {boolean} enabled - Whether or not to enable the DHCP server
+ * @returns {boolean} Boolean indicating success of the command.
+ */
+function setDhcpStatus(enabled) {
+  if (!uciSet('dhcp.lan.ignore', enabled ? '0' : '1')) {
+    return false;
```

Comment:
Do we want to log something in these cases? Otherwise if it fails, we'll have no idea why it failed.

Reply:
I figured we'd log wherever these functions get called.